src/apurva/NumPrint.java

```diff
@@ -0,0 +1,50 @@
+package apurva;
+
+class NumPrint {
+	void eveNum(int start, int end) {
+		int index = start;
+		while (index <= end) {
```

Comment:
here you need to define one extra variable, you can handle this 
if(start<=end){
//code
}
start++;